packages/ui-drilldown/src/Drilldown/index.tsx

```diff
@@ -1377,6 +1377,10 @@ class Drilldown extends Component<DrilldownProps, DrilldownState> {
           getDisabledOptionProps
         }) => (
           <View
+            themeOverride={{
```

Comment:
we shouldn't use `themeOverride` for our own components

Reply:
removed